src/pybind/mgr/cephadm/module.py

```diff
@@ -393,6 +394,12 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
             default=10,
             desc='max number of osds that will be drained simultaneously when osds are removed'
         ),
+        Option(
+            'service_discovery_port',
+            type='int',
+            default=8765,
```

Comment:
Is this getting opened? On my test where firewalld was enabled, 9000 was not enabled during bootstrap

Reply:
fixed in the last push